packages/cashier/src/modules/withdrawal-email-verification/withdrawal-email-verification.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { MobileWrapper } from '@deriv/components';
+import { useVerifyEmail } from '@deriv/hooks';
+import { useStore, observer } from '@deriv/stores';
+import { localize, Localize } from '@deriv/translations';
+import { isCryptocurrency } from '@deriv/shared';
+import RecentTransaction from '../../components/recent-transaction';
+import EmailVerificationEmptyState from '../../components/email-verification-empty-state';
+import EmptyState from '../../components/empty-state';
+import Error from '../../components/error';
+import ErrorStore from '../../stores/error-store';
+import { useCashierStore } from 'Stores/useCashierStores';
+
+const WithdrawalEmailVerification = observer(() => {
+    const verify = useVerifyEmail('payment_withdraw');
+    const { client } = useStore();
+    const { transaction_history } = useCashierStore();
+
+    React.useEffect(() => {
+        transaction_history.onMount();
+    }, [transaction_history]);
+
+    if (verify.error) return <Error error={verify.error as ErrorStore} />;
+
+    if (verify.has_been_sent) return <EmailVerificationEmptyState type={'payment_withdraw'} />;
+
+    return (
+        <>
+            <EmptyState
+                icon='IcCashierAuthenticate'
+                title={localize('Please help us verify your withdrawal request.')}
+                description={
+                    <>
+                        <Localize i18n_default_text="Hit the button below and we'll send you an email with a link. Click that link to verify your withdrawal request." />
+                        <br />
+                        <br />
+                        <Localize i18n_default_text='This is to protect your account from unauthorised withdrawals.' />
+                    </>
+                }
+                action={{
+                    label: localize('Send email'),
+                    onClick: verify.send,
+                }}
+            />
+            <MobileWrapper>{isCryptocurrency(client.currency) && <RecentTransaction />}</MobileWrapper>
```

Comment:
Can we move `RecentTransaction` out of `WithdrawalEmailVerification` and handle it in `withdrawal.tsx` using `PageContainer`?